### PR TITLE
edit/createRow Default Values

### PR DESCRIPTION
**Pull request by benrr101**
Returns strings for the default value of a column when a new row is created. The values that could come back:
* `null` when there isn't a default for the column
* a string when there is a default for the column
* a placeholder (currently <TBD>) when the column cannot be updated

### CODE REVIEW

File: src/Microsoft.SqlTools.ServiceLayer/EditData/EditColumnWrapper.cs [hunk 1]
```diff
@@ -19,11 +19,23 @@ public class EditColumnWrapper
         public DbColumnWrapper DbColumn { get; set; }
 
         /// <summary>
+        /// If set, this is a string representation of the default value. If set to null, then the
```
Comment (kburtram) [inline src/Microsoft.SqlTools.ServiceLayer/EditData/EditColumnWrapper.cs]: >If set to null [](start = 74, length = 14)

I suppose this logic works fine when the default value is null?

File: src/Microsoft.SqlTools.ServiceLayer/Localization/sr.strings [hunk 2]
```diff
@@ -200,6 +200,8 @@ EditDataScriptFilePathNull = An output filename must be provided
 
 EditDataCommitInProgress = A commit task is in progress. Please wait for completion.
 
+EditDataComputedColumnPlaceholder = <TBD>
```
Comment (kburtram) [inline src/Microsoft.SqlTools.ServiceLayer/Localization/sr.strings]: ><TBD> [](start = 36, length = 5)

We should probably find a better placeholder than this IMO.  Is this what SSDT or SSMS uses?

Comment (benrr101) [inline src/Microsoft.SqlTools.ServiceLayer/Localization/sr.strings]: SSMS - Doesn't show defaults. Just fills all fields in with "NULL".
SSDT - Doesn't show defaults. Just fills all fields in with "NULL".
JetBrains - Uses "<Generated>" as the place holder. Fields with defaults are shown as "<Default>"

Comment (benrr101) [inline src/Microsoft.SqlTools.ServiceLayer/Localization/sr.strings]: I'm open for suggestions for what to use here. Though I think that using "NULL" isn't the best idea.

File: src/Microsoft.SqlTools.ServiceLayer/EditData/EditColumnWrapper.cs (hunk 1 above)
Comment (benrr101) [inline src/Microsoft.SqlTools.ServiceLayer/EditData/EditColumnWrapper.cs]: The logic basically generates a list of display values for the new row in the UI.
* No default is returned as `null`
* `NULL` default is returned as "NULL" (as per how SMO returns the value)
* Literal defaults are unwrapped from what SMO gives us and returned (eg, `((8))` => 8, `N'stuff'` => stuff)
* Computed columns are returned as a placeholder

**Pull request merged**